src/types/wallet/index.ts

```diff
@@ -48,6 +48,7 @@ export type TransactionPartyType = {
   msg: string
   label: string
   address?: string | null
+  timeStamp?: string | null
```

Comment:
No need to type it as `null`. `?` makes it optional (`undefined`). Same for `address`.

Reply:
Will be fixed with #259